cli/cli_util.c

```diff
@@ -52,7 +52,7 @@ rest_kvs_list(const char *socket_path, const char *alias, struct yaml_context *y
     next = buf;
     next[strlen(next) - 1] = '\0'; /* Get rid of the trailing newline char */
 
-    c = strsep(&next, "\n"); /* Advance next past 'kvs_list:' */
+    strsep(&next, "\n"); /* Advance next past 'kvs_list:' */
```

Comment:
The snprint seems unnecessary?  The inner while loop should probably check for the NUL character...

Reply:
This code is going away with the REST PR, so I am fine leaving it as it. I just made these changes to appease clang-tidy.